assets/stylesheets/gu-sass/helpers.scss

```diff
@@ -38,3 +38,16 @@
     }
     @return $string;
 }
+
+// Helper to create the multilines used in e.g. the header.
```

Comment:
Apparently, this file is indented with four spaces and others are with two spaces. Is it worth to convert it to two since there are only 40 lines here?

Reply:
Done.